rootfs-builder/clearlinux/config.sh

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#Use "latest" to always pull the last Clear Linux Release
+OS_VERSION=latest
+PACKAGES="systemd coreutils-bin iptables-bin libudev0-shim"
```

Comment:
This needs to be documented in a bit more detail on what packages we install by default and why. Also we should call out how to use the extra packages arguments.

Reply:
docuemented in config_template.sh https://github.com/kata-containers/osbuilder/pull/1/files#diff-c0cf277570d2f9b6da2f327d33611047R12,

Removed coreutils from required packages.